packages/peaclock/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/octobanana/peaclock
+TERMUX_PKG_DESCRIPTION="A responsive and customizable clock, timer, and stopwatch for the terminal."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@efreak"
+TERMUX_PKG_VERSION=0.4.3
+TERMUX_PKG_SRCURL=${TERMUX_PKG_HOMEPAGE}/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=3de32486e45ddf6a62c591619b5373d23c8eaf885975441ae581ca7241c48012
+TERMUX_PKG_DEPENDS="libicu"
+TERMUX_PKG_BUILD_DEPENDS="cmake"
```

Comment:
Can be removed, cmake is setup by [configure step automatically](https://github.com/termux/termux-packages/blob/master/scripts/build/configure/termux_step_configure.sh#L9)

Reply:
This is useful for on-device builds though, which is [why I added these build dependencies for the Swift package](https://github.com/termux/termux-packages/blob/master/packages/swift/build.sh#L11), none of which are needed when cross-compiling that compiler package.

Reply:
I guess we have not fully defined how to deal with build dependencies on device. cmake, pkgconfig, ninja and perl are installed if user runs [./scripts/setup-termux.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-termux.sh), and for cmake in particular there is a message from [termux_setup_cmake to install it](https://github.com/termux/termux-packages/blob/master/scripts/build/setup/termux_setup_cmake.sh#L28-L34) if it is missing (so I would say there is no reason to specify cmake in TERMUX_PKG_BUILD_DEPENDS)

Reply:
Good point.